### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD043.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#45] [General]
# https://blog.3place.cn/3place/fsd/FSD043.html

https://blog.3place.cn/3place/fsd/FSD043.html

**Comment by picklecai** [comment]
浮士德抢夺战

**Comment by picklecai** [comment]
要每天争取自由和生存的人，
才有享受两者的权利。

不求生就没有生的权利，不求自由就没有自由民的权利。
没有求生欲，虽生犹死。

**Comment by picklecai** [comment]
永恒的创造于我们何补！
被创造的又使它复归于无！

梅菲斯特是说：熵让负熵毫无意义。创造出生物来，生物只会将已有的破坏殆尽。谁愿意天天像西斯弗西斯一样呢？

**Comment by picklecai** [comment]
没想到天使的方法竟然是火烧小鬼，情迷老魔。梅菲斯特后悔莫迭。
